app/org/thp/cortex/util/docker/DockerClient.scala

```diff
@@ -0,0 +1,172 @@
+package org.thp.cortex.util.docker
+
+import com.github.dockerjava.api.model._
+import com.github.dockerjava.core.{DefaultDockerClientConfig, DockerClientConfig, DockerClientImpl}
+import com.github.dockerjava.transport.DockerHttpClient
+import com.github.dockerjava.zerodep.ZerodepDockerHttpClient
+import play.api.{Configuration, Logger}
+
+import java.nio.file.{Files, Path}
+import java.time.Duration
+import java.util.concurrent.{Executors, TimeUnit}
+import scala.concurrent.blocking
+import scala.concurrent.duration.FiniteDuration
+import scala.jdk.CollectionConverters._
+import scala.util.Try
+
+class DockerClient(config: Configuration) {
+  private lazy val logger: Logger           = Logger(getClass.getName)
+  private lazy val (dockerConf, httpClient) = getHttpClient
+  private lazy val underlyingClient         = DockerClientImpl.getInstance(dockerConf, httpClient)
+
+  def execute(containerId: String): Try[Int] =
+    Try {
+      val startContainerCmd = underlyingClient.startContainerCmd(containerId)
+      startContainerCmd.exec()
+      val waitResult = underlyingClient
+        .waitContainerCmd(containerId)
+        .start()
+        .awaitStatusCode()
+      logger.info(s"container $containerId started and awaited with code: $waitResult")
+
+      0
+    } recover {
+      case e =>
+        logger.error(s"execute container $containerId failed", e)
+
+        1
+    }
```

Comment:
If you catch all exception, it is not needed to return a try, it will never fail.
I'd prefer to not catching exception at all.